bin/delete_lambda.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+import json
+
+# Gets user input for the prefix of lambdas to delete
+prefix = input("Enter Prefix: ")
+
+# Gets ORCA lambda functions with given prefix
+get_functions = f"aws lambda list-functions --query 'Functions[] | [?contains(FunctionName, `{prefix}`) == `true`]'"
+completed_process = subprocess.run(get_functions, shell=True, capture_output=True)
+output = completed_process.stdout
+convert = json.loads(output.decode("utf-8").replace("'","'"))
+
+# Deletes ORCA lambda functions with the given prefix
+for sub in convert:
+    print("Deleting " + sub['FunctionName'])
+    lambda_output = sub['FunctionName']
+    delete_function = f"aws lambda delete-function --function-name {lambda_output}"
+    subprocess.run(delete_function, shell=True, capture_output=True)
```

Comment:
I am a bit confused with the script. This looks like it will delete all our lambdas but we only want to delete a few lambdas having ENIs are associated with the `vpc-postgres-ingress-all-egress` security group.

These are the lambdas I see that need to be deleted in my env. 
```
rhassanorca1_internal_reconcile_report_job
rhassanorca1_copy_from_archive
```

Reply:
AWS will just list a single function as "owning" an ENI once you delete those two it will list a new function that needs to be deleted. Found a script that will list all Lambdas associated with an ENI and found that all the ORCA Lambdas were attached
<img width="1336" alt="eni" src="https://github.com/nasa/cumulus-orca/assets/131377950/7217b96c-a02d-4b51-97e8-7b01bf2f4ecb">

Reply:
I can create a shell script that will delete just the ORCA Lambdas so there aren't any accidental deletions of any other Lambdas in a users account with the same prefix aren't deleted